javatests/arcs/core/data/ParticleSpecTest.kt

```diff
@@ -48,6 +69,65 @@ class ParticleSpecTest {
     assertFailsWith<IllegalArgumentException> { spec.dataflowType }
   }
 
+  @Test
+  fun egressType_noEgressType_returnsNull() {
```

Comment:
nit - `egressType_noType_returnsNull` name is more consistent with other tests.

Reply:
I like it.